### PR TITLE
* supported windows path format

### PR DESCRIPTION
**Pull request by kkvskkkk**

### CODE REVIEW

**Comment by LiXizhi** [comment]
are you sure BOOST_ROOT_PATH is working? The doc says to set BOOST_ROOT. Maybe you only need to change the set BOOST_ROOT line

File: Client/CMakeLists.txt [hunk 1]
```diff
@@ -105,23 +105,23 @@ endif()
...
+		set(BOOST_ROOT_PATH ${SERVER_SOURCE_DIR}/boost_1_65_0)
 	elseif(IS_DIRECTORY ${SERVER_SOURCE_DIR}/boost_1_61_0)
-		set(BOOST_ROOT ${SERVER_SOURCE_DIR}/boost_1_61_0)
+		set(BOOST_ROOT_PATH ${SERVER_SOURCE_DIR}/boost_1_61_0)
 	else()
 		message(WARNING "You can define a global environment variable of BOOST_ROOT that specify the boost root dir")
 	endif()
 else()
 	message("BOOST_ROOT env is at: $ENV{BOOST_ROOT}")
-	set(BOOST_ROOT "$ENV{BOOST_ROOT}")
```
Comment (LiXizhi) [inline Client/CMakeLists.txt]: maybe fixing this line is enought

File: Server/trunk/sqlite-3.6.23.1/CMakeLists.txt [hunk 2]
```diff
@@ -21,12 +21,12 @@ include_directories("${NPLRUNTIME_ROOT}/Core")
 # include boost if client is not defined, it allows us to use a different boost version than the client on the server build.
 if (PARAENGINE_CLIENT OR WIN32)
```
Comment (LiXizhi) [inline Server/trunk/sqlite-3.6.23.1/CMakeLists.txt]: this entire code block can be removed from cmake. since both client and server share the same boost

**Comment by kkvskkkk** [comment]
fixed

**Pull request merged**